src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -220,6 +220,17 @@ public bool RunPreviewIsChecked
             }
         }
 
+        /// <summary>
+        /// Controls the Enabled property in the Show Run Preview toogle button
+        /// </summary>
+        public bool RunPreviewEnabled
```

Comment:
Is this property bind to the toggle? I cant tell from this PR

Reply:
Yes, it is bound to the "isEnabled" property on the controller. It seams at some point we removed it by mistake and it was causing one of this issues.